hello/lib/hello_web/endpoint.ex

```diff
@@ -1,9 +1,13 @@
 defmodule HelloWeb.Endpoint do
   use Phoenix.Endpoint, otp_app: :hello
 
-  # The session will be stored in the cookie and signed,
-  # this means its contents can be read but not tampered with.
-  # Set :encryption_salt if you would also like to encrypt it.
+
+
+  socket "/socket", HelloWeb.UserSocket,
+  websocket: true,
+  longpoll: false
```

Comment:
It is used to set up a dynamic comunication between the server and the client. With this function it's possible to receive responses from the server without requesting them. In this case it has a false value because we are using websocket for that purpose.